src/pin-code/pin-code.js

```diff
@@ -34,6 +35,14 @@ export default class PinCode extends React.Component<PropsT, StateT> {
     }
   }
 
+  getMaskStyle(i: number) {
+    if (this.props.values[i] == '') return this.props.values[i];
+    if (typeof this.props.mask !== 'boolean') {
```

Comment:
```suggestion
    if (typeof this.props.mask === 'string') {
```

Reply:
Could also combine these conditionals a bit:
```js
if (this.props.values[i] !== '' && typeof this.props.mask === 'string') {
  return this.props.mask;
} else {
  return this.props.values[i];
}
```

Reply:
Yeah, thanks, this style is much better.